docker/webapp_config.json

```diff
@@ -43,7 +44,10 @@
   },
   "system_settings": {
     "logpath": "/usr/local/homer",
-    "logname" : "homer-app.log"
+    "logname" : "homer-app.log",
+    "_loglevels": "can be: fatal, error, warn, info, debug, trace",
```

Comment:
Ask @adubovikov who wrote that. And IMO YES it's VERY HELPFUL for retards poking around who cannot find appropriate documentation.

Reply:
I never said it was not useful, i just said since the config is already confusing enough we should use `_comment` for comments, instead of baptizing more fields? Is that fair?

Reply:
Disagree.  `_comment_loglevels` seems sane - since it's not clear what `_comment` would 'belong' to otherwise. 

Take this up with @adubovikov since I copy-pasted  ;)

https://github.com/sipcapture/homer-app/commit/d415942a9964bc66e3f0fea71ce5bb4b4317a258